packages/libs/error-stack/README.md

```diff
@@ -95,7 +95,7 @@ fn main() -> Result<(), ExperimentError> {
 
 This will most likely result in an error and print
 
-![](assets/full.png)
+![](https://github.com/hashintel/hash/blob/8ed55bd73045fba83a7ea2e199b31d5b829537b9/packages/libs/error-stack/assets/full.png?raw=true)
```

Comment:
Unsure if we want to use a permalink here, or a path on `main`. Using a permalink means that if we refactor the directory structure, we won't break old links (good).

Using a path means that it will automatically get updated if we push a fix (also good), but will (perhaps silently) get broken if we change the folder structure (bad). Also old versions that have been persisted (for example on `crates.io`) are probably not patchable

Unsure what we want to go with here, cc @nonparibus, I think the patchable argument probably means we should stick with the permalink as is

Reply:
Having a link to the main branch will also update the link on external pages. If we update the image, the image will also be updated on e.g. crates.io

Reply:
Which is what I meant by:
> Using a path means that it will automatically get updated if we push a fix (also good

Reply:
As an immediate fix should we just use the permalink?

Reply:
Yes. Captured a [task](https://app.asana.com/0/1201095311341924/1203084649340060/f) for the full-blown object storage sync to be implemented later.

Reply:
Absolute path intro'd in https://github.com/hashintel/hash/pull/1153 so marking this thread resolved.